From 8fd79203fde0a69c61b554e53bf95db84cc87ced Mon Sep 17 00:00:00 2001
From: chewitt <christianshewitt@gmail.com>
Date: Sat, 19 Sep 2020 16:25:19 +0000
Subject: [PATCH 1/5] HACK: skin.estuary: make event log conditional and move
 LE settings

Move LE settings from the bottom grid to the top panel and hide Event Log
when service.libreelec.settings exists to retain the layout.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 addons/skin.estuary/xml/Settings.xml | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/addons/skin.estuary/xml/Settings.xml b/addons/skin.estuary/xml/Settings.xml
index fa63b174b8..e25d96dec2 100644
--- a/addons/skin.estuary/xml/Settings.xml
+++ b/addons/skin.estuary/xml/Settings.xml
@@ -54,10 +54,17 @@
 						<onclick>ActivateWindow(systeminfo)</onclick>
 						<icon>icons/settings/sysinfo.png</icon>
 					</item>
+					<item>
+						<label>LibreELEC</label>
+						<onclick>RunAddon(service.libreelec.settings)</onclick>
+						<icon>icons/settings/libreelec.png</icon>
+						<visible>System.AddonIsEnabled(service.libreelec.settings)</visible>
+					</item>
 					<item>
 						<label>$LOCALIZE[31067]</label>
 						<onclick>ActivateWindow(eventlog)</onclick>
 						<icon>icons/settings/eventlog.png</icon>
+						<visible>!System.HasAddon(service.libreelec.settings)</visible>
 					</item>
 				</content>
 			</control>
@@ -111,12 +118,6 @@
 						<onclick>ActivateWindow(SystemSettings)</onclick>
 						<icon>icons/settings/system.png</icon>
 					</item>
-					<item>
-						<label>LibreELEC</label>
-						<onclick>RunAddon(service.libreelec.settings)</onclick>
-						<icon>icons/settings/libreelec.png</icon>
-						<visible>System.AddonIsEnabled(service.libreelec.settings)</visible>
-					</item>
 				</content>
 			</control>
 		</control>
-- 
2.17.1

